src/views/Benchmark/index.jsx

```diff
@@ -35,6 +39,31 @@ class Benchmark extends Component {
     this.mounted = false;
   }
 
+  // eslint-disable-next-line class-methods-use-this
+  handleTooltipClick(e, items) {
+    const globalChartObj = Chart;
+    // check if the click happened on empty chart space or data point
```

Comment:
I suggest not leaving so many comments like this in the code because it makes it more cluttered and most are unnecessary. For future work, I'd suggest making code as self-explanatory as possible with how you name variables and functions (I think you did fine with naming for this feature though). For blocks of code that have the potential to be confusing or complex, try to think about whether a comment would be important to have months from now if you had to look at the code again.